From 49d6872853c0c3a54d1bff02ef6bdf7099a97166 Mon Sep 17 00:00:00 2001
From: TheShermanTanker <tanksherman27@gmail.com>
Date: Fri, 18 Aug 2023 18:00:01 +0800
Subject: [PATCH] bootstrap-lto config should use -flto=auto

---
 config/bootstrap-lto.mk | 10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

diff --git a/config/bootstrap-lto.mk b/config/bootstrap-lto.mk
index 1ddb1d870ba..be8b5d5fe23 100644
--- a/config/bootstrap-lto.mk
+++ b/config/bootstrap-lto.mk
@@ -1,10 +1,10 @@
 # This option enables LTO for stage2 and stage3 in slim mode
 
-STAGE2_CFLAGS += -flto=jobserver -frandom-seed=1
-STAGE3_CFLAGS += -flto=jobserver -frandom-seed=1
-STAGEprofile_CFLAGS += -flto=jobserver -frandom-seed=1
-STAGEtrain_CFLAGS += -flto=jobserver -frandom-seed=1
-STAGEfeedback_CFLAGS += -flto=jobserver -frandom-seed=1
+STAGE2_CFLAGS += -flto=auto -frandom-seed=1
+STAGE3_CFLAGS += -flto=auto -frandom-seed=1
+STAGEprofile_CFLAGS += -flto=auto -frandom-seed=1
+STAGEtrain_CFLAGS += -flto=auto -frandom-seed=1
+STAGEfeedback_CFLAGS += -flto=auto -frandom-seed=1
 
 # assumes the host supports the linker plugin
 LTO_AR = $$r/$(HOST_SUBDIR)/prev-gcc/gcc-ar$(exeext) -B$$r/$(HOST_SUBDIR)/prev-gcc/
-- 
2.35.1.windows.2

